packages/module/patternfly-docs/content/extensions/extended-components/examples/DetailsPage.md

```diff
@@ -0,0 +1,81 @@
+---
+# Sidenav top-level section
+# should be the same for all markdown files
+section: extensions
+subsection: react-component-groups
+# Sidenav secondary level section
+# should be the same for all markdown files
+id: DetailsPage
+# Tab (react | react-demos | html | html-demos | design-guidelines | accessibility)
+source: react
+# If you use typescript, the name of the interface to display props for
+# These are found through the sourceProps function provided in patternfly-docs.source.js
+propComponents: [
+  'DetailsPage', # No output --> see https://github.com/patternfly/patternfly-org/issues/3423
```

Comment:
I ran in to a number of issues trying to get props documented.  I believe this is the optimal combination of config for the most docs.  But it is not exhaustive of all props given the issues.  See https://patternfly.slack.com/archives/C4FM977N0/p1682601405903069